src/nih_wayfinding/app/index.html

```diff
@@ -92,7 +92,24 @@ <h2 class="navbar-actionheader" ng-cloak ui-sref="profiles">{{ navbar.config.tit
     <script src="bower_components/angular-leaflet-directive/dist/angular-leaflet-directive.js"></script>
     <script src="bower_components/Leaflet.encoded/Polyline.encoded.js"></script>
     <script src="bower_components/turf/turf.min.js"></script>
+    <script src="bower_components/leaflet-plugins/control/Distance.js"></script>
```

Comment:
Assuming these being added is left over from when @ddohler added this dependency. Add an ignore to the Gruntfile for everything we don't need, which is everything except layer/tile/Google.js. The gruntfile should have an example already, likely in bootstrap.